Sources/SwiftSyntaxBuilder/gyb_generated/BuildableNodes.swift

```diff
@@ -12522,20 +12250,6 @@ public struct TupleType: TypeBuildable, ExpressibleAsTupleType {
     assert(rightParen.text == ")")
   }
 
-  /// A convenience initializer that allows:
-  ///  - Initializing syntax collections using result builders
-  ///  - Initializing tokens without default text using strings
-  public init(
-    leftParen: TokenSyntax = TokenSyntax.`leftParen`,
-    rightParen: TokenSyntax = TokenSyntax.`rightParen`,
-    @TupleTypeElementListBuilder elementsBuilder: () -> ExpressibleAsTupleTypeElementList = { TupleTypeElementList([]) }
```

Comment:
Please keep this as `TupleTypeElement` conforms to `HasTrailingComma`
The same for the `FunctionType`

Reply:
✅ I agree that using a result builder here makes sense because a `TupleType` contains multiple types by design.

Reply:
I didn’t expect us to be using `TupleTypeElementList` for `FunctionType` as well. In that case I agree with you that it’s better to not have `TupleTypeElementList` be result builder type.